apps/website/package.json

```diff
@@ -1,60 +1,46 @@
   "private": true,
+  "type": "module",
   "dependencies": {
-    "@amazeelabs/bridge-gatsby": "^1.2.7",
-    "@amazeelabs/cloudinary-responsive-image": "^1.6.15",
+    "@amazeelabs/bridge-waku": "^1.1.3",
     "@amazeelabs/decap-cms-backend-token-auth": "^1.1.7",
-    "@amazeelabs/gatsby-plugin-operations": "^1.1.3",
-    "@amazeelabs/gatsby-plugin-static-dirs": "^1.0.1",
-    "@amazeelabs/gatsby-source-silverback": "^1.14.0",
     "@amazeelabs/publisher": "^2.4.30",
     "@amazeelabs/strangler-netlify": "^1.1.9",
     "@amazeelabs/token-auth-middleware": "^1.1.1",
     "@custom/cms": "workspace:*",
-    "@custom/decap": "workspace:*",
     "@custom/schema": "workspace:*",
     "@custom/ui": "workspace:*",
-    "gatsby": "^5.13.1",
-    "gatsby-plugin-layout": "^4.13.0",
-    "gatsby-plugin-manifest": "^5.13.0",
-    "gatsby-plugin-netlify": "^5.1.1",
-    "gatsby-plugin-pnpm": "^1.2.10",
-    "gatsby-plugin-robots-txt": "^1.8.0",
-    "gatsby-plugin-sharp": "^5.13.0",
-    "gatsby-plugin-sitemap": "^6.13.0",
-    "gatsby-plugin-uninline-styles": "^0.2.0",
-    "gatsby-source-filesystem": "^5.13.0",
-    "image-size": "^1.1.1",
-    "mime-types": "^2.1.35",
-    "netlify-cli": "^17.21.1",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "netlify-cli": "^17.29.0",
+    "react": "19.0.0-rc.0",
+    "react-dom": "19.0.0-rc.0",
+    "react-error-boundary": "^4.0.13",
+    "react-server-dom-webpack": "19.0.0-rc.0",
+    "waku": "0.21.0-alpha.2"
   },
   "devDependencies": {
     "@netlify/edge-functions": "^2.3.1",
     "@netlify/functions": "^2.6.0",
     "@testing-library/react": "^14.1.2",
-    "@types/react": "^18.2.46",
-    "@types/react-dom": "^18.2.18",
+    "@types/react": "^18.3.3",
+    "@types/react-dom": "^18.3.0",
     "@types/serve-static": "^1.15.5",
     "happy-dom": "^12.10.3",
     "start-server-and-test": "^2.0.3",
     "vitest": "^1.1.1"
   },
   "scripts": {
     "test:static": "tsc --noEmit && eslint '**/*.{ts,tsx,js,jsx}' --ignore-path='./.gitignore'",
-    "full-rebuild": "pnpm clean && pnpm build:gatsby",
+    "full-rebuild": "pnpm clean && pnpm build:waku",
     "start:drupal": "pnpm run --filter @custom/cms start",
-    "build:drupal": "CLOUDINARY_CLOUDNAME=test pnpm start-test start:drupal 8888 build:gatsby",
-    "build:gatsby": "gatsby build",
-    "build": "if node has-drupal.mjs; then pnpm build:drupal; else pnpm build:gatsby; fi",
+    "build:drupal": "CLOUDINARY_CLOUDNAME=test pnpm start-test start:drupal 8888 build:waku",
+    "build:waku": "waku build",
+    "build": "if node has-drupal.mjs; then pnpm build:drupal; else pnpm build:waku; fi",
     "start": "publisher",
-    "serve": "netlify dev --cwd=. --dir=public --port=8000",
+    "serve:proxy": "mitmproxy -p 8000 --mode reverse:http://localhost:8002/",
+    "serve:proxied": "netlify dev --cwd=. --dir=dist/public --port=8002",
+    "serve": "netlify dev --cwd=. --dir=dist/public --port=8000",
     "dev": "pnpm clean && publisher",
```

Comment:
The `clean` command is missing here I guess - Phillip responded that `it has not cache right now, so there is not point to clear`, so perhaps remove it?